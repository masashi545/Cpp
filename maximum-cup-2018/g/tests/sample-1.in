[([(1)o][(2)x]??][([(?)x][(4)o])x]
4
2 0 0 1
