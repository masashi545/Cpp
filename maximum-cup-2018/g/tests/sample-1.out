[([(1)o][(2)x])o][([(3)x][(4)o])x]
